Documentation/HALFunctionReferences.adoc

```diff
@@ -0,0 +1,3 @@
+# HAL Driver References
+
+* For further documentation for HAL Drivers (e.g. CAN, ADC, GPIO) http://www.disca.upv.es/aperles/arm_cortex_m3/llibre/st/STM32F439xx_User_Manual/index.html[HAL Drivers]
```

Comment:
This shows up as

`For further documentation for HAL Drivers (e.g. CAN, ADC, GPIO) HAL Drivers`, did you mean to remove the first `HAL Drivers`?